src/hooks/useBreakpoint.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { BreakpointContext } from '../contexts/BreakpointContext';
+
+export const useBreakpoint = <T extends any>(values: T[]): T => {
+	if (values.length > 4) {
```

Comment:
Same as in the provider, we should be more flexible in accepting a range of array sizes and dynamically handle it (for example between 1 and 6)